blog/2024/rippled-2.2.1.md

```diff
@@ -0,0 +1,72 @@
+---
+category: 2024
+date: 2024-06-30
```

Comment:
@ximinez I think you marked the date as June instead of July here

Reply:
Ugh! Woops. Sorry. Since it's already merged, I'll open a new PR, I guess.

Man, I can not tell you how many times I have mixed up July for June this year. It's weird.